hooks/post/30-upgrade.rb

```diff
@@ -31,13 +31,15 @@ def import_subscriptions
 end
 
 def elasticsearch_message
-  return true unless Kafo::Helpers.execute('rpm -q elasticsearch')
-
+  if `rpm -q elasticsearch`.gsub!(/\A"|"\Z/, '') != nil
```

Comment:
What's this supposed to do? It seems to always be nil.  Using elasticearch (not installed) and coreutils (installed) as an example:

```
irb(main):009:0> `rpm -q elasticsearch`
=> "package elasticsearch is not installed\n"
irb(main):010:0> `rpm -q elasticsearch`.gsub!(/\A"|"\Z/, '')
=> nil
irb(main):007:0> `rpm -q coreutils`.gsub!(/\A"|"\Z/, '')
=> nil
irb(main):008:0> `rpm -q coreutils`
=> "coreutils-8.22-18.el7.x86_64\n"
```